submissions/mod-comparison/mod-comparison.md

```diff
@@ -0,0 +1,106 @@
+
+### Industrial Revolution 2
+
+The next mod on my list is [Industrial Revolution 2](https://mods.factorio.com/mod/IndustrialRevolution). The intent behind this mod is that you work through a series of different tech levels as you progress – starting off with burner and steam power initially, then moving on to normal electrical systems, then unlocking more advanced machines that operate using lasers. We got off to a good start with this mod, having to run steam pipes to all the assembly machines and inserters was an interesting challenge and something that made a nice change to how we were used to thinking about things. Once we moved to the electrical age, deciding when it was worth pulling up the old, steam powered machines to replace them with electrical ones was also interesting, and provided us with an interesting upgrade path that was a bit more involved than the normal “drag an upgrade planner over everything” method from vanilla. However, unfortunately, whilst it was interesting thinking about the upgrades in a few areas, needing to do them across large areas of the factory did get rather repetitive.
+
+![Steam powered assembly machines](IRsteam.png)
+
+Another interesting addition was the larger number of forms each type of metal could take. Unlike vanilla, which basically just has two (plates and either gears or wire), in IR2 most metals can be in ingot, plate, rod, gear, rivet, beam or piston, which means either a lot more rows on your bus, or a lot more processing when you need to make something. We did get used to this system eventually, but I have to admit that I felt like a lot of it was complications for the sake of making the game harder, and making it take longer. 
+
+![Lots of different components!](IRparts.png)
+
+Metal processing is done in quite an interesting way – much like AngelBobs, there are various extra steps you can add in to make the process more efficient, but some of the later ones produce byproducts. In this case though, these byproducts are actually vital in some areas, and can’t be made in any other way so you need to balance it quite carefully. Fortunately, this was much less painful than in AngelBobs, making it an interesting puzzle, rather than a major frustration. I think this was well designed. 
+
+![Sorting gold ore](IRgoldsorting.png)
+
+The additional graphics for this mod are excellently done. Each of the tiers has its own feel, yet it all hangs together very nicely. It really helps to emphasise the progression from steam to laser. 
+
+Over all, I feel that there are some very interesting ideas in this pack – moving through different stages of technology is a fascinating idea with a lot of potential, however unfortunately the fiddliness of organising a lot of the forms of the metals made some of the builds feel like a bit of a chore. Whilst I enjoyed the Industrial Revolution run, I was quite glad to move on once it was finished, and don’t think I’ll be playing it again. 
+
+### Space Exploration
```

Comment:
I think you can mention here that this mod is made by Earendel. Who's also been on the scene for a very long time with this vision, which started with Alien Biomes. He is also known for creating cool looking graphics, and did it so well that he was hired by Wube as Concept Artist!